config/metrician.yaml

```diff
@@ -37,6 +37,16 @@
   :route_tracking:
     :enabled: false
 
+  # "apdex" controls the recording of metrics to measure apdex
+  # More here: https://en.wikipedia.org/wiki/Apdex
+  # Note this requires "request" tracking to be enabled
+  :apdex:
+    :enabled: true
+
+    # set the threshold, in ms, above which a request is no longer
+    # considered "sasifactory". The "tolerates" threshold will be
+    # 4x this number.
+    :satisfied_threshold: 2500
```

Comment:
Since `I.time` measures in seconds, and I think the majority of the time we measure things in seconds, we should either make this seconds, or change the name to `satisfied_threshold_ms`. How's that sound?